duckduckgo/Android
# Escape device app regex queries

Task/Issue URL: https://app.asana.com/0/361428290920652/1164307657927321

**Description**:
Escapes query section of regex to ensure that special characters from user do not invalidate our regex syntax.

**Steps to test this PR**:
1. Add the widget to your home screen
1. Tap on the widget and search for a regex special character e.g "("
1. This should return no results (it previously crashed)

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
